FluentAssertions.Core/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -101,7 +101,7 @@ public class EquivalencyAssertionOptions : EquivalencyAssertionOptionsBase<Equiv
         public EquivalencyAssertionOptions()
         {
             IncludingNestedObjects();
-            IncludingAllDeclaredProperties();
+            IncludingAllDeclaredMembers();
```

Comment:
Yeah, I think you're right after all. I didn't realize that we were talking about the public fields only. That's what you get when you read those discussions between the other work or during breakfast :-)